src/main/java/com/afrunt/jach/validate/ACHValidationResult.java

```diff
@@ -0,0 +1,18 @@
+package com.afrunt.jach.validate;
+
+/**
+ * @author ilyakharlamov
+ *
+ */
+public class ACHValidationResult {
```

Comment:
You know, I thought about validations, but it is very hard to create some common approach due to fact, that each vendor uses ACH in it's own custom manner, so I decided to not include it to the library